CMakeLists.txt

```diff
@@ -54,7 +54,9 @@ endif()
 # Setup the oneDPL library target
 ###############################################################################
 add_library(oneDPL INTERFACE)
-target_compile_features(oneDPL INTERFACE cxx_std_17)
+if(NOT _cmake_windows_compid_workaround)
+    target_compile_features(oneDPL INTERFACE cxx_std_17)
+endif()
```

Comment:
Could you apply the same workaround in https://github.com/oneapi-src/oneDPL/blob/main/cmake/templates/oneDPLConfig.cmake.in?

Reply:
Yes, thanks.  Done.